### PR TITLE
added food.html

### PR DESCRIPTION
**Pull request by arnoldsmithson**

### CODE REVIEW

**Review by akattine — APPROVED** [review]
Looks good

File: topics/food.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Movies</title>
```
Comment (sleddog) [inline topics/food.html]: title still says movies

File: topics/food.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <meta name="description" content="">
+    <title>Foods</title>
+</head>
+<body>
+    <h1>Favorite Foods</h1>
+   
+    <ul>
+		<li> Arnold's favorite food is Curry!</li>
+		<li></li>
+		<li></li>
```
Comment (sleddog) [inline topics/food.html]: empty list elements?  remove em

File: topics/food.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Foods</title>
+</head>
+<body>
+    <h1>Favorite Foods</h1>
+   
+    <ul>
+		<li> Arnold's favorite food is Curry!</li>
```
Comment (sleddog) [inline topics/food.html]: For this, just put in Curry (perhaps specify what type... green)

via github we can determine who added this line... so attribution is stored there

**Review by sleddog — APPROVED** [review]

**Pull request merged**